src/main/scala/com/lucidchart/open/relate/SqlResult.scala

```diff
@@ -195,15 +198,15 @@ class SqlResult(val resultSet: java.sql.ResultSet) {
   def strictURLOption(column: String): Option[URL] = getResultSetOption(resultSet.getURL(column))
 
   protected[relate] def extractOption[A](column: String)(f: (Any) => A): Option[A] = {
-    resultSet.getObject(column).asInstanceOf[Any] match {
-      case x if (x == null || resultSet.wasNull()) => None
+    resultSet.getObject(column) match {
+      case x if resultSet.wasNull => None
```

Comment:
They're equivalent. Though actually, it's probably best (most performant) to remove the other one instead.

> If the value is an SQL NULL, the driver returns a Java null